NEWS.md

```diff
@@ -1,6 +1,18 @@
-## v1.2.0 IN-PROGRESS
+## v2.0.0 2022-08-16
 
-* Supports users interface version 15.3 16.0 (MODINREACH-303)
+Release primarily addresses bug fix.
```

Comment:
sam as in request for master, no idea what it means in scope of 2.0.0, we have features here, not only bugfixes